packages/angular/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1102.3",
+    "@angular-devkit/build-angular": "^13.3.7",
```

Comment:
I think this is supposed to be made on canary?

Reply:
+1, would like @wlee221 to look at this change

Reply:
Also, I don't think we have officially tested Angular 13 yet, so I'm concerned about what changes this would make.